ui/ui-showcase.jsx

```diff
@@ -511,6 +515,42 @@
       return (prefix || "") + this.props.summary.toLowerCase().replace(/\s/g, "-");
     },
 
+    componentDidMount: function() {
```

Comment:
Please add comments describing why we're doing this, as well as more detail on why we have to use setTimeout.  JSDoc comments might be nice.